bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/internal/VoiceManagerImpl.java

```diff
@@ -556,31 +580,37 @@ public void listenAndAnswer() throws IllegalStateException {
 
     @Override
     public void listenAndAnswer(@Nullable STTService stt, @Nullable TTSService tts,
-            @Nullable HumanLanguageInterpreter hli, @Nullable AudioSource source, @Nullable AudioSink sink,
+            @Nullable Collection<HumanLanguageInterpreter> hlis, @Nullable AudioSource source, @Nullable AudioSink sink,
             @Nullable Locale locale, @Nullable String listeningItem) throws IllegalStateException {
         // use defaults, if null
         STTService sttService = (stt == null) ? getSTT() : stt;
         TTSService ttsService = (tts == null) ? getTTS() : tts;
-        HumanLanguageInterpreter interpreter = (hli == null) ? getHLI() : hli;
+        Collection<HumanLanguageInterpreter> interpreters = hlis;
+        if (hlis == null) {
+            HumanLanguageInterpreter hli = getHLI();
+            if (hli != null) {
+                interpreters = List.of(hli);
+            }
```

Comment:
`interpreters` should be set to null if there is no default:
```
interpreters = hli == null ? null : List.of(hli);
```

Reply:
already initialized here:
```
List<HumanLanguageInterpreter> interpreters = hlis;
```

Reply:
Yes, you're right because hlis is null in this case.
Comment to be ignored.